src/urllib3/util/retry.py

```diff
@@ -185,6 +185,9 @@ class Retry:
     #: Maximum backoff time.
     BACKOFF_MAX = 120
 
+    # Backward compatibility; assigned outside of the class.
+    DEFAULT: "Retry"
```

Comment:
It just occurred to me, that this actually needs to be `DEFAULT: ClassVar["Retry"]`, since it's defined once on the class, not on every instance. I forgot about it, sorry!

Reply:
Done!